README.rst

```diff
@@ -157,6 +157,41 @@ Lost mode is slightly different to the "Play Sound" functionality in that it all
 >>> message = 'Thief! Return my phone immediately.'
 >>> api.iphone.lost_device(phone_number, message)
 
+Find My Friends
+==============
+
+Once you have successfully authenticated, you can find your friends' locations!
+
+Locations
+********
+
+Returns the your friends' last known locations. You must install and initialize the Find My Friends app on an iOS device, signing in with your credentials. In addition, your friends must be sharing their location with your account using the Find My Friends app.
+
+>>> api.friends.locations
+
```

Comment:
From #265 PR : Note: To get a specific contact's device location, cycle through all the locations and select the device location where the contact_id=location['id']

Reply:
Or maybe it can be nice to add this function : `api.friends.location(contact_id)` :+1:

Reply:
Yes. I'm aware of this. And yes, that would be a nice convenience function to add.